internal/pkg/service/appproxy/http/router.go

```diff
@@ -354,6 +354,10 @@ func (r *Router) authProxyConfig(app DataApp, provider options.Provider) (*optio
 		},
 	}
 
+	// Cannot separate errors from info because when ErrToInfo is false (default),
+	// oauthproxy keeps forcibly setting its global error writer to os.Stderr whenever a new proxy instance is created.
+	v.Logging.ErrToInfo = true
```

Comment:
> After some consideration I think the errors from oauthproxy aren't actually all that important because they'll either prompt an error in our code or are okay to ignore so putting them into info level in our logger seems fine.

Yes

---------------

It is ok for now, may in future we will fix it better, in the fork.